clusters/build-clusters/01_cluster/prow/03_deployment/admin_config_updater_rbac.yaml

```diff
@@ -206,3 +206,40 @@ items:
   - kind: ServiceAccount
     name: config-updater
     namespace: ci
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: config-updater
+    namespace: openshift-monitoring
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - secrets
+      verbs:
+        - get
+        - create
+        - update
+        - patch
+    - apiGroups:
+        - monitoring.coreos.com
+      resources:
+        - alertmanagers
+        - prometheuses
+      verbs:
```

Comment:
do we need also `create` here?

Reply:
`Create` is done by the prometheus operator.
We maybe not need this at all (will be sure after testing @stevekuznetsov comment above)

Reply:
Not relevant anymore.